app/src/fennecProduction/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  sharedUserId: This release type is meant to replace Firefox (Release channel) and therefore needs to inherit
+                its sharedUserId for all eternity. Shipping an app update without sharedUserId can have
+                fatal consequences. For example see:
+                - https://issuetracker.google.com/issues/36924841
+                - https://issuetracker.google.com/issues/36905922
+-->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+      android:sharedUserId="org.mozilla.firefox.sharedID">
```

Comment:
Same question about Beta and Nightly.  Consider using a placeholder variable here.  The XML file opts you into to `sharedUserId`, but then you set the user ID itself from `build.gradle` (like a version code or similar).